app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -74,7 +74,7 @@ if [ x$ID = xrhel ] || [ x$ID = xcentos ]; then
     yum localinstall $CONSUMER_RPM -y
     rm -f $CONSUMER_RPM
 
-    subscription-manager register --org=<%= @organization.label %> --activationkey="<%= @activation_key %>"
+    subscription-manager register --org=<%= @organization.label %> --activationkey='<%= @activation_key %>'
```

Comment:
Just to be sure I'd also add quotes around the org label. Not sure if it can have spaces, but better safe than sorry.
```suggestion
    subscription-manager register --org='<%= @organization.label %>' --activationkey='<%= @activation_key %>'
```

Reply:
makes sense, added